drake_bazel_external/tools/bazel.rc

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2016-2018, Toyota Research Institute.
```

Comment:
Should be either 2016 or 2018, whenever the particular file was created. Since it appeared in this repo in 2018 and has mostly boilerplate, I would argue for 2018.